tests/shipping/models/Delivery - Second Purchase - Without Geolocation and PostalCode - Boleto.model.js

```diff
@@ -20,11 +20,6 @@ export default function test(account) {
       fillEmail(email)
       confirmSecondPurchase()
       payWithBoleto()
-
-      cy.get('#shipping-data')
-        .contains('21')
-        .should('be.visible')
-
```

Comment:
Can we fix it without deleting it? Also, 21 is the number of days to make the delivery, correct?

Reply:
(I don't know should be the fix though :/)

Reply:
There is not a proper way of testing it. 😢  That is why this model was the only one checking it.

Reply:
And yes, 21 is the number of days to make the delivery.

Reply:
what if we test in a range like `21 <= x <= 26` and put a comment explaining this? So if we ever get a bug where delivery days are very unexpected (ex: 0, trash value, etc), we can catch it.

Reply:
@wdsrocha hold that thought and let's merge without it so we avoid surprises on Black Friday but for sure we must find a better way to validate delivery.

Reply:
Ok, fair enough

Reply:
@wdsrocha I'm not quite sure about that. What are we trying to test in this case? If we selected the right SLA? If so (and I think it is), we already test that by checking the id of the selected SLA. But if the intention is to just check if the estimate for the delivery is right, these tests shouldn't be responsible for that, IMO. For me, this should be done by the ones (logistics?) who provide this SLA.

Reply:
We could do the same as in the shipping-preview tests, where we look for the ids of the selected SLAs to see if it's what we expect.